config/newrelic.yml

```diff
@@ -1,10 +1,9 @@
-<%= Rails.env %>:
-  app_name: <%= Rails.application.secrets.newrelic['app_name'] %>
-  license_key: <%= Rails.application.secrets.newrelic['license_key'] %>
-  developer_mode: <%= Rails.application.secrets.newrelic['developer_mode'] || 'false' %>
-  log_level: <%= Rails.application.secrets.newrelic['log_level'] || 'info' %>
-  monitor_mode: <%= Rails.application.secrets.newrelic['monitor_mode'] ? 'true' : 'false' %>
-  ssl: true
+development:
+  app_name: USA Search
+  license_key: FILL_IN_LICENSE_KEY
+  developer_mode: true
```

Comment:
@noremmie , per the NR docs, "Developer mode is removed and no longer supported as of Ruby agent version 4.1.0."

https://docs.newrelic.com/docs/agents/ruby-agent/features/developer-mode

`monitor_mode: true` should be the only necessary line to enable the dev monitoring. That's how I tested locally.

I'm also thinking we might want to set the dev app name to something like `USA Search - DEVELOPMENT` or even `USA Search - YOUR NAME` to avoid any confusion when looking at the metrics.